poll/public/html/survey.html

```diff
@@ -0,0 +1,55 @@
+{{ js_template|safe }}
+<div class="poll-block" data-private="{% if private_results %}1{% endif %}" data-can-vote="{% if can_vote %}1{% endif %}">
+    {# If no form is present, the Javascript will load the results instead. #}
+    {% if not choices or private_results %}
+    <h3 class="poll-header">{{block_name}}</h3>
+    <form>
+        <table class="survey-table">
+            <thead>
+              <tr>
+                  <th></th>
+                  {% for answer, label in answers %}
+                    <th class="survey-answer">{{label}}</th>
+                  {% endfor %}
+              </tr>
+            </thead>
+        {% for key, question in questions %}
+            <tr class="survey-row">
+                <td class="survey-question">
+                    {% if question.img %}
+                        <div class="poll-image-td">
+                                <img src="{{question.img}}" />
+                        </div>
+                    {% endif %}
+                  {{question.label|safe}}
+                </td>
+            {% for answer, answer_details in answers %}
+                <td class="survey-option">
+                    <input type="radio" name="{{key}}" value="{{answer}}"{% if question.choice == answer %} checked{% endif %}/>
```

Comment:
Yes something like this would work:
```<label><input type="radio" name="{{key}}" value="{{answer}}"{% if question.choice == answer %} checked{% endif %}/><span class="sr">Yes</span></label>```